### DISCUSSION TITLE
How to make event listener touchstart passive

### DISCUSSION THREAD

**Discussion by aroncatti** [sachinchoolur/lightGallery#1231] [General]
I noticed the following warning: 
[Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive. 

I do not find the way to make the touchstart and touchmove passive. Can you help me?
My code is very simple

lightGallery(document.getElementById('ad-videos'), {
    plugins: [lgVideo],
    download: false,
    zoomFromOrigin: false,
    mousewheel: true,
    autoplayVideoOnSlide: true,
    licenseKey: 'xxxxx',
	exThumbImage: 'data-poster',

});
Thank you.. I will appreciate any help.

**Comment by sachinchoolur** [comment]
Hey @aroncatti,

Sorry for the delayed response. This has to be updated from the lightGallery side.

But, we still need to call preventDefault in some cases, and you can not set the events passive. But, I'll try to improve this if possible

**Comment by jgujgu** [comment]
I received this warning as well. Obviously it's not breaking.
